### ISSUE TITLE
The <seriesInfo> elements in <reference> are not kept

### ISSUE THREAD

**Issue by robUx4** [mmarkdown/mmark#113]
For example this XML code after the `{backmatter}`

```xml
<reference anchor="ISO.9899" target="https://www.iso.org/standard/57853.html">
  <front>
    <title>Information technology -- Programming languages -- C</title>
    <author>
      <organization>International Organization for Standardization</organization>
    </author>
    <date month="" year="2011" />
  </front>
  <seriesInfo name="ISO/IEC" value="9899:2011" />
</reference>
```

will produce

```xml
<reference anchor="ISO.9899" target="https://www.iso.org/standard/57853.html">
  <front>
    <title>Information technology -- Programming languages -- C</title>
    <author>
      <organization>International Organization for Standardization</organization>
    </author>
    <date year="2011"></date>
  </front>
</reference>
```

**Comment by miekg** [comment]
[ Quoting <notifications@github.com> in "[mmarkdown/mmark] The <seriesInfo> ..." ]
>For example this XML code after the {backmatter}

hmm, yeah it's a bug.

Note that's because the original xml3 spec is not leading anymore. Basically whatever
xml2rfc implements is the current spec, which makes it impossible to be compliant. And
IIRC seriesInfo has been discussed a lot.

**Comment by miekg** [comment]
closes #113
